source/draft_api.rst

```diff
@@ -223,7 +258,9 @@ Response body:
       "issue_uri":"https://github.com/osu-cass/whats-fresh-api/issues/56",
       "date_worked":null,
       "created_at":2014-09-18,
-      "updated_at":null
```

Comment:
Trying to modify ``created_at`` would require going through and modifying the whole audit trail (``created_at`` is constant in the trail for the "same" object). And I don't really see a benefit to modifying them. They're metadata, if they can be edited they on some level become meaningless.